plugin-server/src/main/ingestion-queues/kafka-queue.ts

```diff
@@ -134,8 +132,7 @@ export class KafkaQueue implements Queue {
         }
     }
 
-    isPaused(targetTopic?: string, partition?: number): boolean {
-        targetTopic = targetTopic ?? this.ingestionTopic
+    isPaused(targetTopic: string, partition?: number): boolean {
```

Comment:
Is partition still optional?

Reply:
Yeah - can pause the whole topic or just a partition